### DISCUSSION TITLE
Admin should be able to display all registered devices

### DISCUSSION THREAD

**Discussion by airdogvan** [Freika/dawarich#315] [Ideas]
In my humble opinion, if only just to check that devices are registering and displaying correctly admin should be able to display all registered devices.  Otherwise admin has to login on each different user's profile to check.

I know this is originally thought for timeline import or gathering.  But can also be used for tracking position.  I use it with Owntrack and Dawarich receives the points frequently enough to effectively track position.  An option to toggle track display would be useful in cases like this.  And also an option to update position on screen without reloading page would be very useful.  Again, I'm perfectly conscious that this is not the original intent but since the functionality is already included, why not make it more practical to use?

To sum up, my suggestions:
1- admin can display all registered devices on screen
2- toggle track (route trace) on/off
3- Refresh latest position without reloading page.

But otherwise many thanks for this project very well done.  For me incredibly useful as I was using Traccar so far, but with the big caveat that when restarting the docker stack all data was lost, although drive was mapped.  Apparently has to do with how they configured the sql driver...
